compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -555,7 +570,7 @@ static LogicalResult
 setVectorDistributionConfig(mlir::FunctionOpInterface entryPoint,
                             linalg::LinalgOp linalgOp,
                             const TargetInfo &targetInfo) {
-  if (isMatmulOrBatchMatmul(linalgOp)) {
+  if (linalg::isaContractionOpInterface(linalgOp)) {
```

Comment:
This does not filter out matvec like operations? So may need some associated logic change in the config function.

Reply:
ah right, good catch

Reply:
Ah actually after some more thought, this wasn't matching on matvec by chance because we don't have any intrinsics for matvec. I added an additional guard for it to be sure.